docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,210 @@
+
+Since July 1st 2024, PMM updates are distributed exclusively through the dedicated `repo.percona.com/pmm2-client` repository. 
+
+To smoothly transition to this repository and verify that your system is correctly configured, download and run our automated script from GitHub: [check_percona_packages.py](https://raw.githubusercontent.com/Percona-Lab/release-aux/main/scripts/check_percona_packages.py).
+
+For detailed instructions and more information about this change, check out [our recent blog post](https://www.percona.com/blog/ensure-the-correct-repositories-are-enabled-for-percona-packages/) and [updated installation instructions](../setting-up/index.md).
+
+#### Update process for older PMM versions
+
+With the migration of PMM’s base image from CentOS 7 to Oracle Linux 9 last year, users running PMM version 2.38 or earlier may experience issues when updating PMM to the latest version using the UI **Update** button.
+
+To avoid these issues and ensure you continue receiving updates:
+
+- **For Docker instances**: perform a one-time Docker update to migrate your installation to the Oracle Linux 9-based image.
+- **For Non-Docker Instances (OVF/AMI)**: follow the upgrade instructions outlined in our [PMM Modernization blog post](https://www.percona.com/blog/pmm-modernized-enterprise-grade-foundation).
+
+### Experimental ARM support for PMM Client
+
+PMM 2.43 introduces experimental support for ARM architecture in PMM Clients, responding to the growing adoption of ARM in data centers and cloud environments. 
+
+For this, PMM now includes pre-built ARM binaries for key features like `node_exporter` and `mysqld_exporter`, enabling monitoring of ARM-based infrastructure while maintaining compatibility with existing PMM Server installations.
+
+As an experimental feature, ARM support will be refined and stabilized in future releases, paving the way for unified monitoring across x86 and ARM architectures.
+
+#### Upgrading and getting started
+
+If you have been compiling PMM on ARM manually, you can now upgrade to the 2.43 release for official support. The [Set up PMM Client topic](../setting-up/client/index.md) now also includes ARM-specific instructions for new installations.
+
+## Maintenance: QAN Update and PostgreSQL 9.x support changes
+
+As part of our ongoing maintenance efforts, we have upgraded the `pg_query_go` package to version 5. This latest version helps parse PostgreSQL queries in QAN, provides improved functionality, and enables subsequent upgrades of important dependencies.
+
+However, with this change, QAN will no longer be able to list PostgreSQL database tables for versions 9.4, 9.5, and 9.6. Since all PostgreSQL 9.x versions have already reached their end-of-life status, we recommend upgrading to a supported version of PostgreSQL.
+
+If you're looking to upgrade, you can easily [install the latest version of Percona Distribution for PostgreSQL](https://docs.percona.com/postgresql/16/installing.html).
```

Comment:
Resisting...temptation...to pitch...our professional services team!

Nope, can't resist.  We should not be afraid to steer people to the commercial side of Percona.  Need help?  We have experts that can execute an upgrade for you...link to sales inquiry.